iframify.js

```diff
@@ -113,8 +30,7 @@
   function getIframeContentForNode (node, extraCSS) {
     return '<!doctype html>' +
       '<html lang="en">' +
-      '<head>' +
-      '<style>' + getStylesForTree(node) + '</style>' +
+      '<head>' + stylesheets +
```

Comment:
I am wondering whether the meta viewport should be added. I guess that’s a topic for another PR though.